recipe-server/client/control/routes.js

```diff
@@ -36,7 +36,7 @@ export default (
           ]}
         />
         <Route
-          path="revision/:revisionId"
+          path="revision/:revisionId/"
```

Comment:
I noticed the bug only appeared when there was _not_ a trailing slash in the URL. Every URL on the site is set up with a trailing slash, and our `absolute-path` util was resolving incorrectly as a result.

I've corrected the route here, as well as another file which was pushing to a non-slash'd route, in order to maintain conformity across the site. However, since the issue actually lied within the `absolute-path` util, the real remedy was to either fix the util or change it out with something else.

After looking for alternative npm packages we could use, I realized we could just use the `resolve` method from node's `path` module! I've swapped that in and deleted the `absolute-path` util all together.